bin/martinize2

```diff
@@ -967,6 +989,17 @@ def entry():
     elif args.cystein_bridge != "auto":
         vermouth.AddCysteinBridgesThreshold(args.cystein_bridge).run_system(system)
 
+    if go:
+        system = vermouth.MergeAllMolecules().run_system(system)
+        # need this here because have to get contact map at atomistic resolution
+        if go_file is None:
+            LOGGER.info("Generating Go model contact map.", type="step")
+            GenerateContactMap(write_file=args.go_write_file).run_system(system)
+        else:
+            LOGGER.info("Reading Go model contact map.", type="step")
+            read_go_map(system=system, path=go_file)
```

Comment:
So, something like this
```suggestion
    if args.go:
        system = vermouth.MergeAllMolecules().run_system(system)
        # need this here because have to get contact map at atomistic resolution
        if isinstance(args.go, str):
            LOGGER.info("Reading Go model contact map.", type="step")
            read_go_map(system=system, path=args.go)
        else:
            LOGGER.info("Generating Go model contact map.", type="step")
            GenerateContactMap(write_file=args.go_write_file).run_system(system)
```